### DISCUSSION TITLE
Customs sound's not working?

### DISCUSSION THREAD

**Discussion by Halroc** [adamk33n3r/runelite-watchdog#162] [Q&A]
Whenever i try to use a custom sound " i have a import for loot drops that uses this *  i can't seem to get the sound to work and it just defaults to the default sound ding. Is there something im missing to make these sounds work?

**Comment by adamk33n3r** [comment]
I'm not sure what the symbols in your message are referring to, but you can check your runelite logs to see if there's an error. It's likely a format/codec issue. Try converting it to mp3 or wav
